data/database.py

```diff
@@ -1414,6 +1418,19 @@ class Meta:
         )
 
 
+class UploadedBlob(BaseModel):
+    """
+    UploadedBlob tracks a recently uploaded blob and prevents it from being GCed
+    while within the expiration window.
+    """
+
+    id = BigAutoField()
+    repository = ForeignKeyField(Repository)
+    blob = ForeignKeyField(ImageStorage)
+    uploaded_at = DateTimeField(default=datetime.utcnow)
+    expires_at = DateTimeField(index=True)
```

Comment:
Should `expires_at` have a default value? e.g. `now + 24 hours`

Reply:
I explicitly made sure it didn't, since if it wasn't specified then row creation will fail on the client, and I want that to be the case :)

Reply:
Smart thinking! Resolving